Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -783,6 +792,8 @@ void GameListCtrl::RescanList()
   // Post UI event to update the displayed list
   QueueEvent(new wxCommandEvent(DOLPHIN_EVT_REFRESH_GAMELIST));
 
+  post_status(_("Scan complete!"));
```

Comment:
Point taken. It still stays there longer than it should, though.

(It doesn't seem to be cleared for me when starting a game, but it does get cleared when exiting a game... Not that it really matters.)